team/team.go

```diff
@@ -0,0 +1,43 @@
+package team
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/astronomer/astro-cli/houston"
+	"github.com/astronomer/astro-cli/pkg/printutil"
+	"github.com/sirupsen/logrus"
+)
+
+// retrieves a team and all of its users if passed optional param
+func Get(args []string, usersEnabled bool, client houston.ClientInterface, out io.Writer) error {
+	teamID := args[0]
```

Comment:
The way I see this kind of `controller` package is that it should be independant from the interface used by the user (CLI entrypoint in this case, it could be GRPC server with Protobuf or HTTP Rest API).

I this method should just directly receive teamID string instead of args []string

Reply:
Also, we might want to validate this teamID before sending a request to the backend, for example check if empty string